tools/influx_si/.shed.yml

```diff
@@ -0,0 +1,4 @@
+categories: [Metabolomics]
+description: metabolic flux estimation based on [in]stationary labeling
+name: influx_si
+owner: sokol
```

Comment:
that should be `workflow4metabolomics`

Reply:
Thanks @bgruening for reviewing.
However, here I have a small question. Which one should be `workflow4metabolomics`: `owner` or `categories`?

Reply:
`owner`